tests/safety/common.py

```diff
@@ -175,6 +174,53 @@ def test_accel_actuation_limits(self, stock_longitudinal=False):
           self.assertEqual(should_tx, self._tx(self._accel_msg(accel)))
 
 
+class LongitudinalGasBrakeSafetyTest(PandaSafetyTestBase, abc.ABC):
```

Comment:
Nice, I wanted to eventually do this, but I forgot if there was a snag making it generic. Feel free to do this in another PR before or after this one

Reply:
once i get this merged I'll look at cleaning up the safety tests for the other brands too